src/auro-flight-main.js

```diff
@@ -45,34 +51,53 @@ class AuroFlightMain extends LitElement {
     `;
   }
 
-  // function that renders the HTML and CSS into  the scope of the component
+  readTime(time) {
+    const timeDigits = time.split(' ')[0];
+    const timeFormat = time.split(' ')[1];
+
+    if (timeFormat === 'am') {
+      return `${timeDigits + ` a m`}`
+    } else {
+      return time
+    }
+  }
```

Comment:
Does this improve how the text read out on the screen reader you're using? I tested the following markup in VoiceOver (iOS) and NVDA and it was all read out as expected ("A M" instead of "am"). Are you seeing something different?

```html
<h1>start</h1>
<p>7:30 am</p>
<p>7:30 AM</p>
<p>7:30AM</p>
<p>7:30am</p>
```

If this is something that makes a difference, we can't rely on the time containing a space. In one of my team's sites, we pass time like "7:30am", not "7:30 am".

Reply:
Interesting. I will test that again. When using the data from the flight UI, `7:30 am` it was read back as `seven thirty am` as in 'I am going...'

Reply:
Here's the CodePen I tested with: https://codepen.io/geoff-rich/pen/popNJKK

Reply:
code removed per new updates